### PR TITLE
Mock SWT/eclipse.ui

### PR DESCRIPTION
**Pull request by soerendomroes**
Adds optional package dependencies to swt and eclipse.ui in klighd and klighd.lsp that can be mocked using the new standalone.mockup plugin.

TODOs
- [ ] wait for ELK deeclipsed version
- [ ] revert change to target platform
- [x] KlighD has to be migrated to EPL-2.0 to use the class and interface stubs.

### CODE REVIEW

**Comment by sailingKieler** [comment]
To be honest: I'm a bit confused. What are rationale and expected benefit of these massive changes?
`klight.standalone.ui` makes no sense to me as there is no UI

**Comment by soerendomroes** [comment]
> To be honest: I'm a bit confused. What are rationale and expected benefit of these massive changes?

See the mail from 7.7.21 titled "Klighd Deeclipsifizierung".
TLDR:
Sometimes we need Klighd without Eclipse UI or SWT, therefore, we mock SWT and eclipse.ui in that case.
This allows us to build an OS-independent diagram language server.

> `klight.standalone.ui` makes no sense to me as there is no UI

There are UI dependencies introduced by piccolo but feel free to suggest a different name.

**Comment by NiklasRentzCAU** [comment]
This idea and code has now been moved to a separate repository, see [SWT-Mock](https://github.com/kieler/SWT-Mock/). Thanks!
